grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -243,7 +244,7 @@ class DocumentTemplateService {
                 requestorMonthlyDemand  : demand?.monthlyDemand ?: 0,
                 requestorQuantityOnHand : quantityOnHand,
                 quantityRequested       : requisitionItem?.quantity,
-                quantityIssued          : requisitionInstance?.isPending() ? "" : requisitionItem?.quantityIssued
+                quantityIssued          : requisitionItem.isApproved() && !requisitionInstance?.isPending() ? requisitionItem?.quantityIssued : ""
```

Comment:
this could be encapsulated in a method like this ...
```
RequisitionItem.isQuantityAllocated()
```
or 
```
RequisitionItem.isQuantityIssued()
```
depending on which one is more accurate.

Reply:
Done